devops/nightly.yml

```diff
@@ -80,7 +91,7 @@ jobs:
     displayName: 'CMake'
     inputs:
       cmakeArgs: |
-        .. -GNinja -DCMAKE_BUILD_TYPE=$(BuildType) -DVERONA_EXTRA_CMAKE_ARGS="-DCMAKE_C_COMPILER=$(CC);-DCMAKE_CXX_COMPILER=$(CXX);-DCMAKE_CXX_FLAGS=$(CXXFLAGS)" -DENABLE_ASSERTS=ON -DUSE_ASAN=$(Asan) -DVERONA_CI_BUILD=On -DRT_TESTS=ON
+        .. -GNinja -DCMAKE_BUILD_TYPE=$(BuildType) -DVERONA_EXTRA_CMAKE_ARGS="-DCMAKE_CXX_FLAGS=$(CXXFLAGS)" -DENABLE_ASSERTS=ON -DUSE_ASAN=$(Asan) -DVERONA_CI_BUILD=On -DRT_TESTS=ON -DVERONA_DOWNLOAD_LLVM=$(LLVMDownload)
```

Comment:
`CXX_FLAGS` should also have been propagated in the same way. We shouldn't need `EXTRA_ARGS` for anything trivial.

Reply:
Should CXX_FLAGS be consistent across LLVM and Verona?

Reply:
That's a good question. Probably not consistently, but if the user adds the option on the top-level project, I'd say "yes".

It depends on what semantics do we want for those CMake flags. I'd assume "top-level flags propagate across all sub-projects" and "project-specific flags go straight to them, and override top-level flags". But that's easier said than done.

Reply:
I like your idea on the other question, let's go with that.